editoast/fga_migrations/src/client.rs

```diff
@@ -0,0 +1,62 @@
+use clap::Args;
+use clap::Parser;
+use clap::Subcommand;
+use fga::client::DEFAULT_OPENFGA_MAX_CHECKS_PER_BATCH_CHECK;
+use fga::client::DEFAULT_OPENFGA_MAX_TUPLES_PER_WRITE;
+use url::Url;
+
+#[derive(Debug, Parser)]
+pub struct CliParser {
+    /// OpenFGA server URL
+    #[arg(long, env = "EDITOAST_OPENFGA_URL", default_value_t = Url::parse("http://localhost:8091").unwrap())]
+    pub openfga_url: Url,
+    /// OpenFGA server max tuple reads per batch check
+    #[arg(long, env = "OPENFGA_MAX_CHECKS_PER_BATCH_CHECK", default_value_t = DEFAULT_OPENFGA_MAX_CHECKS_PER_BATCH_CHECK)]
+    pub tuple_reads: u32,
+    /// OpenFGA server max tuples per write request
+    #[arg(long, env = "OPENFGA_MAX_TUPLES_PER_WRITE", default_value_t = DEFAULT_OPENFGA_MAX_TUPLES_PER_WRITE)]
+    pub tuple_writes: u64,
+    /// OpenFGA migrations store
+    #[arg(long, env = "MIGRATIONS_STORE_NAME", default_value = "fga-migrations")]
+    pub migrations_store: String,
+    /// The store on which to apply the migrations
+    #[arg(
+        long,
+        env = "AUTHORIZATION_STORE_NAME",
```

Comment:
```suggestion
        env = "EDITOAST_AUTHORIZATION_STORE_NAME",
```

we're trying to prefix our environment variables (makes the service they apply to more explicit + easier to `env | grep`)